jbmc/src/java_bytecode/java_bytecode_language.cpp

```diff
@@ -777,15 +774,21 @@ bool java_bytecode_languaget::generate_support_functions(
   convert_lazy_method(res.main_function.name, symbol_table);
 
   // generate the test harness in __CPROVER__start and a call the entry point
-  return
+  if(
```

Comment:
Why is this `return` replaced by an `if`? Unless I'm missing something this code is `if(x) return true; return false;` - which should be the same as `return x;`